source/reactor/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+project(reactor VERSION ${prplmesh_VERSION})
+
+# Set the base path for the current module
+set(MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
+
+# Sources
+file(GLOB_RECURSE reactor_sources ${MODULE_PATH}/src/*.c*)
```

Comment:
I don't like the `GLOB` option. It seems that even CMake itself doesn't like it either:
[Note We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate. The CONFIGURE_DEPENDS flag may not work reliably on all generators, or if a new generator is added in the future that cannot support it, projects using it will be stuck. Even if CONFIGURE_DEPENDS works reliably, there is still a cost to perform the check on every rebuild.](https://cmake.org/cmake/help/latest/command/file.html)